mer_verify_kernel_config

```diff
@@ -172,6 +173,7 @@ CONFIG_EXT4_FS			y,m,!	# Mer uses ext4 as rootfs by default
 CONFIG_FANOTIFY			y,!	# optional, required for systemd readahead.
 CONFIG_INOTIFY_USER		y	# systemd: http://cgit.freedesktop.org/systemd/systemd/commit/README?id=713bc0cfa477ca1df8769041cb3dbc83c10eace2
 CONFIG_IPV6			y,m,!	# systemd: http://cgit.freedesktop.org/systemd/systemd/tree/README#n37
+CONFIG_MODULES			y,m,!   # Required to build kernel modules (Such as WLAN, Memnotify etc)
```

Comment:
Allowing CONFIG_MODULES to be built as module (the 'm' in the allowed list) makes no sense at all.

Reply:
@mlehtima You are right. I misread LKDDB for CONFIG_MODULES. I've updated it now.

Reply:
@mlehtima Are you happy with the changes now?

Reply:
Ultimate nitpicking, the # doesn't align with the surrounding comments.

Reply:
> Ultimate nitpicking, the # doesn't align with the surrounding comments because its indentation is spaces and not tab.

Fixed that now, but there are others in the file that have the same issue. I'm happy to PR those cleanups once this one is merged.

Reply:
Yes, the file has many formatting issues and those can be fixed later if we want. At least not adding any new formatting issues is good.

Reply:
@mlehtima Can we merge this now?

Reply:
Modules are used on a lot of ports to various OSes so the comment should just mention the flag is needed for module support.